### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by 97thFloor**
Optimizations for SEO — primary KW: cloud engineering.

### Proposed changes

Added keyword instances and relevant language based on semantic analysis.

### CODE REVIEW

File: content/blog/building-a-future-of-cloud-engineering/index.md [hunk 1]
```diff
@@ -54,12 +54,12 @@ At the same time, "DevOps" as an approach to organizing teams and
 developing software has matured greatly, and is now the default for many
 organizations. New practices such as infrastructure-as-code, immutable
 infrastructure, and Site Reliability Engineering have modernized these
-roles, giving us approaches that can leverage more rigorous software
-engineering practices to tasks that used to be more ad-hoc, manual, and
+roles, giving us approaches that can leverage a more rigorous software
+engineering experience to tasks that used to be more ad-hoc, manual, and
```
Comment (cnunciato) [inline content/blog/building-a-future-of-cloud-engineering/index.md]: This doesn't quite say the same thing.

File: content/blog/building-a-future-of-cloud-engineering/index.md [hunk 2]
```diff
@@ -83,18 +83,18 @@ approach.
...
-SRE; it's the best of all of them. In this new future, our teams will
+SRE; cloud engineering is the best of all of them. In this new future, our teams will
 experience more joy, collaboration, and productivity than ever before.
 
 Enter Pulumi.
 
 Using your favorite general purpose programming language -- whether it
 is JavaScript, TypeScript, Python, Go, or something else -- you can
-express the entirety of your cloud software. You can develop it in your
+express the entirety of your cloud software. As a cloud engineer, you can develop it in your
```
Comment (cnunciato) [inline content/blog/building-a-future-of-cloud-engineering/index.md]: This paragraph doesn't quite work as well with these additions, IMO.

File: content/blog/building-a-future-of-cloud-engineering/index.md [hunk 3]
```diff
@@ -112,8 +112,8 @@ deployment statuses and to organize and find information more easily.
 It's never been such a breeze to go from idea to production.
 
 We've become enamored by this new approach to creating software that is
-truly born in the cloud. But it's not just our own belief -- it is
-working terrifically in practice. We have worked with organizations of
+truly born in the cloud. But it's not just our own belief -- this new age of cloud engineering is
+working terrifically in practice. We have worked with training and coaching in organizations of
```
Comment (cnunciato) [inline content/blog/building-a-future-of-cloud-engineering/index.md]: `We have worked with training and coaching` doesn't quite make sense.

File: content/blog/building-a-future-of-cloud-engineering/index.md (hunk 1 above)
Comment (97thFloor) [reply content/blog/building-a-future-of-cloud-engineering/index.md]: That one is probably fine to leave as it is then. Unless "can leverage a better experience through more rigorous software engineering practices..." works? It's not a super crucial one, but "experience" kept coming up in our semantic analysis.

File: content/blog/building-a-future-of-cloud-engineering/index.md (hunk 2 above)
Comment (97thFloor) [reply content/blog/building-a-future-of-cloud-engineering/index.md]: Research was showing that we were a little bit under average for variations of "cloud engineer/ing" compared with our SERP competitors. This seemed a natural place to add it in, so it'd be great if we could keep even just a couple instances, or even just the first one.

File: content/blog/building-a-future-of-cloud-engineering/index.md (hunk 3 above)
Comment (97thFloor) [reply content/blog/building-a-future-of-cloud-engineering/index.md]: Those were topics that came up in our semantic analysis, but if the phrasing is too awkward it's fine to leave those out and shouldn't matter too much.

**Comment by zchase** [comment]
PR now tracked in https://github.com/pulumi/docs/pull/2516. Closing.

**Pull request closed without merging**